expanalysis/results.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 24 16:03:04 2016
+
+@author: ian
```

Comment:
Got it!

On Monday, March 28, 2016, Vanessa Sochat <notifications@github.com> wrote:

> In expanalysis/results.py
> <https://github.com/expfactory/expfactory-analysis/pull/6#discussion_r57655934>
> :
>
> > @@ -0,0 +1,166 @@
> > +# -*- coding: utf-8 -*-
> > +"""
> > +Created on Thu Mar 24 16:03:04 2016
> > +
> > +@author: ian
>
> You should replace these headers (from your IDE?) with something akin to
> what we have for the other modules
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/expfactory/expfactory-analysis/pull/6/files/3892e3b7f763661f207c7a1035e5a1b105749329#r57655934>
>


-- 
Best,
Ian